pkg/actions/addon/addon.go

```diff
@@ -34,18 +35,18 @@ type StackManager interface {
 	UpdateStack(stackName, changeSetName, description string, templateData manager.TemplateData, parameters map[string]string) error
 }
 
-func New(clusterConfig *api.ClusterConfig, clusterProvider *eks.ClusterProvider, stackManager StackManager, withOIDC bool, oidcManager *iamoidc.OpenIDConnectManager, clientSet kubeclient.Interface) (*Manager, error) {
+func New(clusterConfig *api.ClusterConfig, clusterProvider eksiface.EKSAPI, stackManager StackManager, withOIDC bool, oidcManager *iamoidc.OpenIDConnectManager, clientSet kubeclient.Interface) (*Manager, error) {
```

Comment:
can we change the var name here as well?

Reply:
Oops! Yes